tests/post/features/salt_api.feature

```diff
@@ -12,7 +12,10 @@ Feature: SaltAPI
     Scenario: Login to SaltAPI using a ServiceAccount
         Given the Kubernetes API is available
         When we login to SaltAPI with the ServiceAccount 'storage-operator'
-        Then we can invoke '["disk.dump", "state.sls"]' on '*'
+        Then we can ping all minions
+        And we can invoke '[".*"]' on '*'
+        And we have '@wheel' perms
+        And we have '@runner' perms
```

Comment:
Uh, really? Should it?

Reply:
Because I don't think the Storage Operator needs to have `*` perms on Nodes

Reply:
> Should it?
I think no, but since we use Bearer token for storage-operator ... it is.
Since we have no "roles" we can only have every perms or None for the moment

Reply:
Maybe related to https://github.com/scality/metalk8s/issues/1528

Reply:
Hmm yes true we have some rules in master config maybe something can be done.
Honnestly I don't really searched how our auth module work, just did a quick fix to have `groups` for Bearer auth.

Reply:
In master config, we have two "groups". One for node-admins, another for storage-operator. The SA for the Operator shouldn't have permissions that make it end up in the node-admins group.

> Maybe related to #1528

No, that one would be nice as well, but that's about reducing the permissions of the storage-operator group. Here, this test effectively confirms that we have the node-admins group, and that shouldn't be the case.

Reply:
Note: `storage-operator` is not a *group*, it's a SaltAPI *user*. Only `node-admins@` is a *group*.

Reply:
Ok just investigate, it's not really easy just retrieve the id_token from dex. (if you have idea do not hesitate)

So several choice for this PR:
- We test with `storage-operator` token as this specific user has the right perms (see https://github.com/scality/metalk8s/pull/2068/files#r351360140)
- We create a new service account just to do this test
- We not test in this PR as this full auth process will be tested by cypress test in #2042

Reply:
I'm getting confused. Why are we talking about Dex? Anything that runs in the cluster can rely on ServiceAccounts, and that's what `storage-operator` uses (or should use) as well.

See:
https://github.com/scality/metalk8s/blob/development/2.4/storage-operator/deploy/role.yaml
https://github.com/scality/metalk8s/blob/development/2.4/storage-operator/deploy/service_account.yaml

Reply:
Since we use dex we can authenticate without service account using directly dex that will give you a token for a user which is bind in `dex-administrator` cluster role binding. But I didn't found any simple solution to retrieve this token (and also it's not really part of salt-api stuff)

I know it's a service account that my point, 
So we need to create a new service account for this test or use the storage-operator one.

Reply:
> we need to create a new service account for this test or use the storage-operator one.

I don't understand the problem. In the Gherkin, we have:

```
When we login to SaltAPI with the ServiceAccount 'storage-operator'
```

I would then expect that's already what the test does. We shouldn't try to authenticate without a SA from those tests (yet, that's something to do, for sure, but will take some time to implement, with the whole OAuth dance).

Reply:
Yes but storage operator is a special case as this user is define directly in the salt-master it may make sense to have a new SA to test the groups function for bearer auth

Reply:
Ah, right, I think I get it now.

So indeed, there is currently no test to check whether *with a given token* for a user that's *not* `storage-operator` but *does* have the rights to do `*` on `Nodes` would get assigned to the 'right' group (i.e., `node-admins`).

So, yes, maybe creating a SA which then gets a `ClusterRoleBinding` to `system:masters`, then check whether we get the expected access to SaltAPI when using this SA's token would work.

No need for Dex or whatnot, so could (should?) be part of this PR.